### ISSUE TITLE
missing a release notes and release assets for 3.10.3

### ISSUE THREAD

**Issue by chenrui333** [FreeRDP/FreeRDP#11014]
👋 looks like 3.10.3 did not get completed, ie the release notes and release assets are missing. Thanks!

<img width="541" alt="image" src="https://github.com/user-attachments/assets/6fa6fdd7-ff06-48db-a49d-1eff1a0a16ce" />

**Comment by akallabeth** [comment]
@chenrui333 yes, not fully release tested yet and not an official release yet.
will take some time, only got limited computer access during holidays.

**Comment by chenrui333** [comment]
make sense, thanks @akallabeth!

**Comment by akallabeth** [comment]
so, created the release tarballs, closing this then.
